eng/pipelines/runtime.yml

```diff
@@ -212,7 +212,7 @@ extends:
             extraStepsTemplate: /eng/pipelines/coreclr/nativeaot-post-build-steps.yml
             extraStepsParameters:
               creator: dotnet-bot
-              testBuildArgs: nativeaot tree nativeaot
+              testBuildArgs: 'nativeaot tree ";nativeaot;tracing/eventpipe/simpleprovidervalidation;"'
```

Comment:
Do we want macos coverage for an event pipe test? I'm wondering if it makes sense to add the basic event pipe test in linux/macos/windows x64 release instead and leave the debug run as just the nativeaot tests.